Python/instruction_sequence.c

```diff
@@ -149,3 +164,288 @@ PyInstructionSequence_Fini(instr_sequence *seq) {
     PyMem_Free(seq->s_instrs);
     seq->s_instrs = NULL;
 }
+
+/*[clinic input]
+class InstructionSequenceType "_PyInstructionSequence *" "&_PyInstructionSequence_Type"
+[clinic start generated code]*/
+/*[clinic end generated code: output=da39a3ee5e6b4b0d input=589963e07480390f]*/
+
+#include "clinic/instruction_sequence.c.h"
+
+static _PyInstructionSequence*
+inst_seq_create_raw(void)
```

Comment:
What the meaning of the the "raw" suffix?

Reply:
Not sure why I called it that. I'll remove.